lib/bundle.js

```diff
@@ -76,6 +77,101 @@ function extractBundleName(fileName) {
   return path.relative(config.pjson.baseURL, fileName.replace(/\.js$/, '')).replace(/\\/g, '/');
 }
 
+function getModuleName (file) {
```

Comment:
We can simplify this implementation by walking the paths with a current match and not breaking on a match, but rather updating the match if there is a better one. The most specific match should be considered the one with the largest value of `path.split('/').length`.

Apart from that the implementation looks good at first glance. Perhaps the regular expressions can be simplified?